docs/DeveloperGuide.md

```diff
@@ -236,116 +241,50 @@ is below:
 * marktask
 * markorder
 
-
-
-
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### Sort orders by amount feature
-
-#### Implementation
-
-The feature sorts all the orders in the addressbook by their amount in descending order. 
-
-To ensure that the orders can be sorted, both `Order` and its attribute `Amount` implement the `Comparable` interface. 
-Order uses its `id` field to produce the default ordering of the `OrderList`. 
-
-{to be completed}
-
-### Display client's total orders feature
-
-#### Implementation
-
-The feature displays the total orders for each client in a new window. Its mechanism is a mix of the mechanisms for `MainWindow` and `HelpWindow`. 
-
-### \[Proposed\] Undo/redo feature
-
-#### Proposed Implementation
-
-The proposed undo/redo mechanism is facilitated by `VersionedSalesNote`. It extends `SalesNote` with an undo/redo history, stored internally as an `salesNoteStateList` and `currentStatePointer`. Additionally, it implements the following operations:
-
-* `VersionedSalesNote#commit()` — Saves the current address book state in its history.
-* `VersionedSalesNote#undo()` — Restores the previous address book state from its history.
-* `VersionedSalesNote#redo()` — Restores a previously undone address book state from its history.
-
-These operations are exposed in the `Model` interface as `Model#commitSalesNote()`, `Model#undoSalesNote()` and `Model#redoSalesNote()` respectively.
+### Update Person Changes in Order List and Task List
 
-Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
+SalesNote's clients are directly referenced in orders. Any changes in the clients through user commands should be propagated through the Order list.
 
-Step 1. The user launches the application for the first time. The `VersionedSalesNote` will be initialized with the initial address book state, and the `currentStatePointer` pointing to that single address book state.
+* When a client is deleted, his/her orders and the tasks linked to the orders will be deleted as well.
```

Comment:
Just a note - using "their" instead of "his/her" might be simpler.  But this is not really important so feel free to ignore.